x-pack/test_serverless/functional/test_suites/apps/lens/open_in_lens/tsvb/top_n.ts

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../../../ftr_provider_context';
+
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const { lens, timePicker, dashboard } = getPageObjects([
+    'lens',
+    'timePicker',
+    'dashboard',
+  ]);
+
+  const testSubjects = getService('testSubjects');
+  const retry = getService('retry');
+  const filterBar = getService('filterBar');
+  const queryBar = getService('queryBar');
+  const panelActions = getService('dashboardPanelActions');
+  const kibanaServer = getService('kibanaServer');
+
+
+  describe('Top N', function describeIndexTests() {
+    const fixture = 'x-pack/test_serverless/functional/fixtures/kbn_archiver/lens/open_in_lens/tsvb/top_n.json';
+
+    before(async () => {
+      await kibanaServer.importExport.load(fixture);
+    })
+
+    after(async () => {
+      await kibanaServer.importExport.unload(fixture);
+    })
+
+    beforeEach(async () => {
+      await dashboard.goToApp(); // required for svl until dashboard PO navigation is fixed
```

Comment:
Without calling this and returning from a given dashboard, the `gotoDashboardEditMode` method will fail attempting to find and interact with the breadcrumbs. This is a temporary solution for now.